dtdl-tools/build.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+use std::process::Command;
+
+// This build script builds all the .NET projects in this digital_twins_connector folder.
```

Comment:
update comment to reference 'dtdl-tools' instead of 'digital_twins_connector'

Reply:
Fixed.